pkg/virt-handler/node-labeller/cpu_plugin.go

```diff
@@ -189,6 +189,15 @@ func (n *NodeLabeller) getDomCapabilities() (HostDomCapabilities, error) {
 	domCapabilitiesFile := filepath.Join(n.volumePath, n.domCapabilitiesFileName)
 	hostDomCapabilities := HostDomCapabilities{}
 	err := n.getStructureFromXMLFile(domCapabilitiesFile, &hostDomCapabilities)
+	if err != nil {
+		return hostDomCapabilities, err
+	}
+
+	if hostDomCapabilities.SEV.Supported == "yes" && hostDomCapabilities.SEV.MaxESGuests > 0 {
+		hostDomCapabilities.SEV.SupportedES = "yes"
```

Comment:
`SupportedES = "yes"` seems redundant since you already have `MaxESGuests > 0`. What is the reason of introducing `SupportedES`?